src/header/common/cookie.rs

```diff
@@ -154,6 +159,17 @@ impl fmt::Display for Cookie {
     }
 }
 
+#[allow(missing_debug_implementations)]
```

Comment:
I'd go ahead a slap `#[derive(Debug)]` on this instead. It shouldn't actually leak anything about cows, and instead should allow someone to view a list of strings pairs.

Also, I'm surprised the `deny(missing_docs)` lint didn't catch no documentation on this struct...